mysql/hooks/init

```diff
@@ -2,13 +2,11 @@
 
 exec 2>&1
 
-set -eo pipefail
```

Comment:
Think you could leave just the `set -e` part? That would still be helpful if the `mysqld` command failed, as it would cause the `init` hook to fail (which is I think what we want).

Reply:
Done ;)

Reply:
I think we should `set -e` in init as well as run so we fail early (if we actually fail)